front-end/src/MyShows.js

```diff
@@ -48,24 +48,27 @@ const ShowGrid = (props) => {
       setFilteredShows([]);
     }
     else {
-      const res = filterShows(props.shows, props.status).map( (showUserInfo) => {
+      const res = filterShows(props.shows, props.status).map((showUserInfo) => {
         return shows.find(show => show.id === showUserInfo.id);
       });
-      setFilteredShows( res );
+      setFilteredShows(res);
     }
   }, [props.shows, props.status, shows]);
+  
 
   return (
     <>
       <h3>My Shows</h3>
       <div id="show-container">
         {filteredShows !== undefined && filteredShows.length !== 0
           ? filteredShows.map((show) => {
-            return <img src={mockShowImage(show.id)} alt={`cover-${show.id}`} key={show.id} />
+            return <img alt={`cover-${show.id}`} key={show.id} />
```

Comment:
Why was `src={mockShowImage(show.id)}` deleted? It breaks the image display on `MyShows`

Reply:
this was an error, sorry for that